code/game/turfs/simulated.dm

```diff
@@ -55,13 +60,15 @@
 			else
 				wet_overlay = image('icons/effects/water.dmi', src, "wet_static")
 		overlays += wet_overlay
-	if(!infinite)
-		spawn(rand(790, 820)) // Purely so for visual effect
-			if(!istype(src, /turf/simulated)) //Because turfs don't get deleted, they change, adapt, transform, evolve and deform. they are one and they are all.
-				return
-			MakeDry(wet_setting)
+	if(time == INFINITY)
+		return
+	if(time == null)
```

Comment:
Couldn't you just do `if(!timing)` here?

Reply:
I did this one to be compatable with floors being wet for 1 frame. Not sure why it would be used but it can now! :P

Reply:
I'd prefer `!time` personally--`== null` checks are kinda gross  unless it's absolutely required.

Reply:
Changed it in the last commit